.env.example

```diff
@@ -1,4 +0,0 @@
-accessKeyId=ACCESSKEYID
-secretAccessKey=SECRETACCESSKEY
-bucket=bucket
-region=us-east-1
```

Comment:
Is there a reason we're deleting this file?

Reply:
I removed it, because it was written `example`, I thought it was rubbish. Only that.

I can reverse this if you prefer

Reply:
Please do :) `.something.example` files are meant to illustrate what a `.something` should contain, without having to commit secrets, and without having to rename files and such. This way you can `.gitignore` the `.env` file and never accidentally leak secrets to the git repository.

Reply:
Poke @aureliosaraiva